### PR TITLE
Add callback URL and getData() saveData() buttons

### PR DESCRIPTION
**Pull request by scullingworth**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,11 +3,20 @@ const express = require('express');
 const https = require('https');
 const http = require('http');
 const fs = require('fs');
+const bodyParser = require('body-parser');
```
Comment (neil-power-hs) [inline index.js]: looks like this is a pretty standard js library?

File: assets/stream.js [hunk 2]
```diff
@@ -24,6 +24,16 @@ function bindApiButtons() {
     hsp.composeMessage(getSingleElementByClassName('hs_composeMessageInput').value);
   });
 
+  getSingleElementByClassName('hs_saveData').addEventListener('click', function () {
+    hsp.saveData(getSingleElementByClassName('hs_saveDataInput').value, function(){});
+  });
+
+  getSingleElementByClassName('hs_getData').addEventListener('click', function () {
+    hsp.getData(function (data){
+      replaceTextInClass('hs_getDataOutput', data);
```
Comment (neil-power-hs) [inline assets/stream.js]: do we want to log this to the console?

File: index.js (hunk 1 above)
Comment (simba11) [reply index.js]: for express, yep!

File: assets/stream.js (hunk 2 above)
Comment (scullingworth) [reply assets/stream.js]: Right now the way this works is to show it in the UI so anyone can use it right in the stream.

Comment (neil-power-hs) [reply assets/stream.js]: even better!

**Pull request merged**